pandas/core/ops/__init__.py

```diff
@@ -474,11 +475,57 @@ def dispatch_to_series(left, right, func, str_rep=None, axis=None):
     """
     # Note: we use iloc to access columns for compat with cases
     #       with non-unique columns.
+    eval_kwargs = eval_kwargs or {}
+
     import pandas.core.computation.expressions as expressions
 
     right = lib.item_from_zerodim(right)
+
     if lib.is_scalar(right) or np.ndim(right) == 0:
 
+        new_blocks = []
+        mgr = left._data
+        for blk in mgr.blocks:
+            # Reshape for EA Block
+            blk_vals = blk.values
+            if hasattr(blk_vals, "reshape"):
+                # ndarray, DTA/TDA/PA
+                blk_vals = blk_vals.reshape(blk.shape)
+                blk_vals = blk_vals.T
+
+            new_vals = array_op(blk_vals, right, func, str_rep, eval_kwargs)
+
+            # Reshape for EA Block
+            if is_extension_array_dtype(new_vals.dtype):
```

Comment:
For my own understanding: the `if` and the `elif` would be unnecessary if all EAs were allowed to be 2D?

Reply:
Yep.  Everything from 489-524 would boil down to something like:

```
new_vals = array_op(blk.values.T, right, func, str_rep, eval_kwargs)
nb = blk.make_block(new_vals.T)
new_blocks.append(nb)
```